bin/potion

```diff
@@ -18,6 +18,7 @@ class PotionCommandLine
      > potion create model foo
      > potion create screen foo
      > potion create table_screen foo
+     > potion create table_screen_cell bar_cell
```

Comment:
Well, there is a metal cell too, was my thinking. But either way is cool with me.